src/pretix/plugins/sendmail/templates/pretixplugins/sendmail/history.html

```diff
@@ -0,0 +1,51 @@
+{% extends "pretixcontrol/event/base.html" %}
+{% load i18n %}
+{% load bootstrap3 %}
+{% block title %}{% trans "Send out emails" %}{% endblock %}
+{% block content %}
+    <h1>{% trans "Email history" %}</h1>
+    <div>
+        <ul class="list-group">
+            {% for log in logs %}
+                <li class="list-group-item logentry">
+                    <p class="meta">
+                        <span class="fa fa-clock-o"></span> {{ log.datetime|date:"SHORT_DATETIME_FORMAT" }}
+                        {% if log.user %}
+                            <br/><span class="fa fa-user"></span> {{ log.user.get_full_name }}
+                        {% endif %}
+                        {% if log.display %}
+                            <br/><span class="fa fa-comment-o"></span> {{ log.display }}
+                        {% endif %}
+                    </p>
+                    <p>
+                        <strong>Email address:</strong> {{ log.parsed_data.recipients }}
```

Comment:
Darn! You are absolutely right! :laughing: I meant to remove it but.... *sigh*

I looked around and `log.parsed_data` outputs an array of a one letter order status (in this case the recipients), such as `['p', 'c']` and I don't know of a way to convert these to the full order status as 'paid', 'canceled', etc in a decent way. 

I wrote a temporary code to deal with this in the template and display the full order status but it has two big problems: 1- it doesn't support translations and 2- if eventually one added a new order status one would have to fix this part too... :confused: I guess these things are not supposed to be dealt with in the template first place, right?
The code I wrote looks like this:
```
{% for status in log.parsed_data.sendto %}
    {% if status == 'n' %}pending orders,{% endif %}
    {% if status == 'p' %}paid orders,{% endif %}
    {% if status == 'e' %}expired orders,{% endif %}
    {% if status == 'c' %}canceled orders,{% endif %}
    {% if status == 'r' %}refunded orders,{% endif %}
{% endfor %}
```
So, I see three options:
1) remove the recipients altogether
2) use the single letters (this might not be helpful for users...)
3) use the chunk I wrote - well, unless you think of another way to process this

What do you think?